src/ui/settings.js

```diff
@@ -98,4 +98,17 @@ sessionIdInputElement.oninput = async ({srcElement: {value}}) => {
   refreshStashTabs();
 };
 
+const RefreshTimeInputElement = document.getElementById('refresh-time-input');
+RefreshTimeInputElement.value = settings.get('overlay.refreshTime') || 60;
```

Comment:
Here we can reuse the constant

```suggestion
RefreshTimeInputElement.value = settings.get('overlay.refreshTime') || DEFAULT_REFRESH_TIME;
```